Reachability/Monitor.swift

```diff
@@ -75,44 +75,42 @@ public class NetworkStatusSubscription {
     }
     
     deinit {
-        self.monitor?.removeReachabilitySubscription(self)
+        monitor?.removeSubscription(self)
     }
 }
 
 public class Monitor {
     
     /// Returns the current reachability status
-    public var reachabilityStatus: ReachabilityStatus {
-        get {
-            var flags = SCNetworkReachabilityFlags()
-            
-            if SCNetworkReachabilityGetFlags(self.reachabilityReference, &flags) {
-                return ReachabilityStatus.statusForReachabilityFlags(flags)
-            }
+    public var status: ReachabilityStatus {
+        var flags = SCNetworkReachabilityFlags()
             
-            return .Unknown
+        if SCNetworkReachabilityGetFlags(self.reachabilityReference, &flags) {
+            return ReachabilityStatus.statusForReachabilityFlags(flags)
         }
+        
+        return .unknown
     }
     
-    var reachabilitySubscriptions = [NetworkStatusSubscriber]()
-    var unsafeSelfPointer = UnsafeMutablePointer<Monitor>.alloc(1)
-    
-    private let callbackQueue = dispatch_queue_create("com.willowtreeapps.Reachability", DISPATCH_QUEUE_CONCURRENT)
-    private var monitoringStarted = false;
+    var subscriptions = [NetworkStatusSubscriber]()
+    var unsafeSelfPointer = UnsafeMutablePointer<Monitor>.allocate(capacity: 1)
+
+    private let callbackQueue = DispatchQueue(label: "com.willowtreeapps.Reachability", attributes: .concurrent)
+    private var monitoringStarted = false
     
-    var reachabilityReference: SCNetworkReachabilityRef!
+    var reachabilityReference: SCNetworkReachability!
     var reachabilityFlags: SCNetworkReachabilityFlags?
     
     /// Initialize monitoring for general internet connection
     convenience public init?()
     {
         var zeroAddress = sockaddr_in()
-        bzero(&zeroAddress, sizeofValue(zeroAddress))
-        zeroAddress.sin_len = UInt8(sizeofValue(zeroAddress))
+        bzero(&zeroAddress, MemoryLayout.size(ofValue: zeroAddress))
+        zeroAddress.sin_len = UInt8(MemoryLayout.size(ofValue: zeroAddress))
         zeroAddress.sin_family = UInt8(AF_INET)
```

Comment:
Not 100% related to Swift 3 conversion, but does this work in an IPv6 only environment?